lib/domain/dtos/CreateLogDto.js

```diff
@@ -16,14 +16,19 @@ const Joi = require('@hapi/joi');
 const BodyDto = Joi.object({
     title: Joi.string()
         .required()
-        .min(3),
+        .min(3)
+        .max(140),
```

Comment:
I would expect this change to also be reflected in the openapi specification and an e2e test.

Reply:
I've helped John to create an e2e test and updated the openapi spec.